modelbaker/iliwrapper/ili2dbconfig.py

```diff
@@ -130,6 +130,8 @@ def __init__(self):
         self.db_ili_version = None
 
     def append_args(self, args, values, consider_metaconfig=False):
+        # since passing metaconfig is not supported yet. see https://github.com/claeis/ili2db/issues/392 we disable the functionality by setting consider_metaconfig to false
```

Comment:
Maybe worth tracing a warning if consider_metaconfig is set to True?

Reply:
Good point.

Reply:
I did not made a proper trace (by passing the message over a signal to the log panel) and just a print instead. this because it's the only message in this class and will be solved in ili2db soon anyway.